public/app/plugins/panel/logs/types.ts

```diff
@@ -12,6 +13,7 @@ type isFilterLabelActiveType = (key: string, value: string, refId?: string) => P
 type isOnClickShowFieldType = (value: string) => void;
 type isOnClickHideFieldType = (value: string) => void;
 export type onNewLogsReceivedType = (allLogs: DataFrame[], newLogs: DataFrame[]) => void;
+type onLogOptionsChangeType = (option: keyof LogListControlOptions, value: string | boolean | string[]) => void;
```

Comment:
Would be nice if the `onLogOptionsChangeType` was usable in the plugin instead of needing to manually sync these types